cpp/oneapi/dal/algo/pca/backend/cpu/train_kernel_svd.cpp

```diff
@@ -71,13 +71,22 @@ static result_t call_daal_kernel(const context_cpu& ctx,
     const auto daal_variances = interop::convert_to_daal_homogen_table(arr_vars, 1, column_count);
 
     daal_pca::internal::InputDataType dtype = daal_pca::internal::nonNormalizedDataset;
+    //TODO: investigate opportunity to change default behavior
+    if (desc.is_scaled() == true && desc.is_mean_centered() == true) {
```

Comment:
You can implement these checks as a free functions on descriptors

Reply:
Comparing Boolean's with "true" or "false" values is ambiguous

Reply:
Its a temporary workaround for more clear understanding of a different scenarios behavior